<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PieSettings</class>
 <widget class="QWidget" name="PieSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QGroupBox" name="labelGroupBox">
       <property name="title">
        <string>Label</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Shadow Offset</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="labelShadowOffsetSpinBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Opacity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="labelShadowCheckBox">
            <property name="text">
             <string>Shadow</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="Widgets::ColorPushButton" name="labelColorButton">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="labelItalicCheckBox">
            <property name="text">
             <string>Italic</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="labelBoldCheckBox">
            <property name="text">
             <string>Bold</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Background Color</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="Widgets::ColorPushButton" name="labelBgColorButton">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="labelOpacitySpinBox">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="titleGroupBox">
       <property name="title">
        <string>Title</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="6" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Background Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="titleShadowCheckBox">
            <property name="text">
             <string>Shadow</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Opacity</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="Widgets::ColorPushButton" name="titleColorButton">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="Widgets::ColorPushButton" name="titleBgColorButton">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="titleItalicCheckBox">
            <property name="text">
             <string>Italic</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="titleBoldCheckBox">
            <property name="text">
             <string>Bold</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Shadow Offset</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="titleShadowOffset"/>
          </item>
          <item row="4" column="2">
           <widget class="QDoubleSpinBox" name="titleOpacitySpinBox">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="titleEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="legendGroupBox">
       <property name="title">
        <string>Legend</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QSpinBox" name="entriesNumSpinBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Background Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="showScalarCheckBox">
            <property name="text">
             <string>Show Scalar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Padding</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="Widgets::ColorPushButton" name="legendBgColorButton">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="paddingSpinBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="showBorderCheckBox">
            <property name="text">
             <string>Show Border</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Number of Entries:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="showBoxCheckBox">
            <property name="text">
             <string>Show Box</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="useBackgroundCheckBox">
            <property name="text">
             <string>Use Background</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Background Opacity</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="bgOpacitySpinBox">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/colorpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
